packages/@interactjs/utils/domUtils.ts

```diff
@@ -213,16 +213,14 @@ export function getElementClientRect (element: Element): Required<Rect> {
   const clientRect =
     element instanceof domObjects.SVGElement ? element.getBoundingClientRect() : element.getClientRects()[0]
 
-  return (
-    clientRect && {
-      left: clientRect.left,
-      right: clientRect.right,
-      top: clientRect.top,
-      bottom: clientRect.bottom,
-      width: clientRect.width || clientRect.right - clientRect.left,
-      height: clientRect.height || clientRect.bottom - clientRect.top,
-    }
-  )
+  return {
+    left: clientRect.left || 0,
+    right: clientRect.right || 0,
+    top: clientRect.top || 0,
+    bottom: clientRect.bottom || 0,
+    width: clientRect.width || clientRect.right - clientRect.left || 0,
+    height: clientRect.height || clientRect.bottom - clientRect.top || 0,
+  }
```

Comment:
Just to be more clear, you can return a new object immediately if a `clientRect` is undefined.
```suggestion
  if (clientRect == null) {
    return {
      left: 0,
      right: 0,
      top: 0,
      bottom: 0,
      width: 0,
      height: 0,
    };
  }

  return {
    left: clientRect.left,
    right: clientRect.right,
    top: clientRect.top,
    bottom: clientRect.bottom,
    width: clientRect.width || clientRect.right - clientRect.left,
    height: clientRect.height || clientRect.bottom - clientRect.top,
  };
```